### ISSUE TITLE
[Feature]: Get the transparency of the web page

### ISSUE THREAD

**Issue by qiqi233** [MicrosoftEdge/WebView2Feedback#4930]
### Describe the feature/enhancement you need

Can I pick up the transparency of the web page at the mouse position?

### The scenario/use case where you would use this feature

When I use webview2, I embed it in my 3D program (UnrealEngine). I want to realize the function of mouse penetration of web pages. When my mouse picks up the transparency value of 0, all interactions with webview2 will be disabled. Otherwise, I can interact with webview2.

### How important is this request to you?

Critical. My app's basic functions wouldn't work without it.

### Suggested implementation

I hope there is a way to get the transparency of the web page at the current mouse position in real time (for example: GetCurrentMousePositionTransparent), so that I can continue with the following operations

### What does your app do? Is there a pending deadline for this request?

无

**Comment by qiqi233** [comment]
Or can you provide a method that allows me to obtain the buffer color data (RGBA format) of the current web page in real time?

**Comment by sandeepchads** [comment]
Since this ask is critical for you, have you tried injecting JavaScript into the web page to detect the transparency of elements at the mouse position. You can then use web messaging to communicate this information back to your host application. 

Would love to hear your thoughts.

**Comment by qiqi233** [comment]
> My main concern is whether it is universal for all web pages, through JavaScript scripts, sorry I am not very familiar with JavaScript

My main concern is whether it is universal for all web pages, through JavaScript scripts, sorry I am not very familiar with JavaScript

**Comment by yoyomule** [comment]
Yes, having this feature is highly necessary when using WebView2 in programs like UE. We hope the WebView2 team can take this requirement into consideration.

**Comment by qiqi233** [comment]
> 是的，在 UE 等程序中使用 WebView2 时，此功能非常必要。我们希望 WebView2 团队能够考虑到这一要求。 使用 JavaScript 通信解决方案可能会导致鼠标延迟、增加 Web 开发工作量并增加计算开销。因此，我们希望获得原生支持。

There are still too few people in this business demand, and it is difficult for the official to pay attention to it. It is really too difficult.

**Comment by yoyomule** [comment]
Unreal's official side has provided a solution for the webui, but there are problems with poor performance. If webview2 can support the above requirements, it would be perfect to overlay webview2 on top of Unreal, and then block or allow the penetration of mouse events by setting the transparency according to the color, as well as connect the keyboard events with UE.
